packages/shared/src/components/LoginButton.tsx

```diff
@@ -43,7 +48,7 @@ export default function LoginButton({
     <Button
       onClick={onClick}
       icon={icon}
-      className={classNames('btn-primary', className)}
+      className={classNames('btn-primary', className, buttonColor.button)}
```

Comment:
This buttonColor should actually overwrite the btn-primary, else we have contradicting classes.